DirectProgramming/DPC++FPGA/ReferenceDesigns/qri/third-party-programs.txt

```diff
@@ -0,0 +1,253 @@
+oneAPI Code Samples - Third Party Programs File

```

Comment:
I don't think you need this if QRD doesn't show up in this file. Confirm with Mike T, but I think you can delete this file.

Reply:
Tanner is correct, please remove this file.  You can also remove it from QRD if it is there.